plugins/jreleaser-maven-plugin/src/main/java/org/jreleaser/maven/plugin/internal/JReleaserModelConverter.java

```diff
@@ -220,6 +220,14 @@ private static void convertGitService(GitService service, org.jreleaser.model.Gi
         s.setMilestone(convertMilestone(service.getMilestone()));
         s.setConnectTimeout(service.getConnectTimeout());
         s.setReadTimeout(service.getReadTimeout());
+
+        convertGitlabSpecific(service, s);
+    }
+
+    private static void convertGitlabSpecific(GitService service, org.jreleaser.model.GitService s) {
```

Comment:
This method may be removed and its contents added to `convertGitlab()`

Reply:
Ah, there 😅